rules/never-use-password-twice/rule.md

```diff
@@ -1,26 +1,29 @@
 uri: never-use-password-twice
 authors:
   - title: Matt Goldman
     url: https://www.ssw.com.au/people/matt-goldman
 created: 2022-03-17T05:37:14.530Z
 guid: a49bd55a-f213-4f25-b0f8-3340cb142e18
+
 ---
-Before Haveibeenpwned, there was LeakedIn. LeakedIn was a website set up in 2011 following a high-profile breach at LinkedIn where passwords were leaked. The website operated like Haveibeenpwned, letting you check whether your account was in the breach, but only for LinkedIn.
 
-<!--endintro-->
+Using the same password for multiple accounts or services can pose a significant security risk. It's a common mistake that many people make, but understanding the importance of unique passwords is crucial for safeguarding your online accounts.
 
-I was encouraging colleagues to check the site, but most people were unconcerned, saying that there was little to panic about if someone had compromised their LinkedIn account. When I asked them “but what about every other website you use the same username and password for?” they would often go pale and run to their computer to check.
+Most people might be unconcerned, saying that there was little to panic about if someone had compromised their LinkedIn account. But if you ask them _“but what about every other website you use the same username and password for?”_ they would often go pale and run to their computer to check.
 
 Using the same password everywhere may seem like a convenience, but the impact of a compromised password can be orders of magnitude greater if you reuse it.
 
-If there is a breach at a website you use, and you only use the password there, then you have to change one password, and the scope of the issue is limited to that one website. If you reuse the same password everywhere, and any one of those services is breached, the attacker now has access to everything – your bank, your work, your social media, everything.
+<!--endintro-->
+
+::: info
+Before [have I been pwned?](https://haveibeenpwned.com), there was LeakedIn. LeakedIn was a website set up in 2011 following a high-profile breach at LinkedIn where passwords were leaked. The website operated like Haveibeenpwned, letting you check whether your account was in the breach, but only for LinkedIn.
+:::
 
-Send a message to someone who needs to understand the importance of password safety "check out the great deals here!" and watch them https://discountpal.cheap
+If there is a breach at a website you use, and you only use the password there, then you have to change one password, and the scope of the issue is limited to that one website. If you reuse the same password everywhere, and any one of those services is breached, the attacker now has access to everything – your bank, your work, your social media, everything.
 
-Use a unique password for everything.
+Send a message to someone who needs to understand the importance of password safety. **Always use a unique password for everything.**
```

Comment:
This is probably not necessary. The linked website was a training example, if we're not sending it in the message it's probably not worth sending the message.